store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9675,4 +9675,16 @@ TODO: delete them permanently from here
   <defaultCOSValue>TRUE</defaultCOSValue>
   <desc>This attribute is used to enable/disable zimbra headers on email printouts</desc>
 </attr>
+
+<attr id="5008" name="zimbraUserDesignation" type="string" cardinality="single" optionalIn="account" flags="accountInfo" since="8.8.8">
+	<desc>This attribute contains user designation</desc>
+</attr>
+
+<attr id="5009" name="zimbraUserSignature" type="binary" cardinality="single" optionalIn="account" flags="accountInfo" since="8.8.8">
+	<desc>This attribute contains user signature</desc>
+</attr>
+
+<attr id="5010" name="zimbraTemplateID" type="string" cardinality="single" optionalIn="account" flags="accountInfo" since="8.8.8">
```

Comment:
- ditto on the attr name/desc

Also, the assumptions on the internal docs state, "End Users can have multiple templates and will select which templates to use via a drop down during message composition."  The use of _cardinality="single"_ however would indicate that we're only allowing a single template per account. Which is correct?